### PR TITLE
Ignore trim filter for Hook

### PR DESCRIPTION
**Pull request by fabiosvm**
* Ignore trim (Horizontal whitespace) filter;
* Update instructionSet, and
* Fix core module loading

### CODE REVIEW

File: test/compilers/hook-tests.js [hunk 1]
```diff
@@ -59,78 +59,84 @@ describe('Hook compiler', () => {
         const expected = {
             asm: [
                 {
+                    labels: [],
```
Comment (RubenRBS) [inline test/compilers/hook-tests.js:62]: Is this expected to _always_ return empty labels no matter what?¿

**Review by RubenRBS — APPROVED** [review]

**Comment by fabiosvm** [comment]
For the input of this test these labels will always be empty.
I made a big JS object which is expected for the below line not to throw exception.

```js
result.should.deep.equal(expected);
```

I could remove these `labels` fields from the result by tweaking the `HookCompiler#processAsm()` method. What do you think?

**Comment by fabiosvm** [comment]
OK. So I will leave it, with the empty labels array.

**Comment by fabiosvm** [comment]
Oops! parsingTime may be different each time it is executed. I will correct it.

**Review by RubenRBS — APPROVED** [review]

**Comment by RubenRBS** [comment]
Thanks! Sorry for the delay in getting this merged. lgtm so lets wait for the CI and then I'll merge

**Comment by fabiosvm** [comment]
Always in your time. No problem. ;) Thanks.

**Comment by dkm** [comment]
This is now live :rocket:

**Pull request merged**